ngraph/frontend/onnx/onnx_import/include/onnx_import/core/model.hpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <onnx/onnx_pb.h>
+#include <ostream>
+#include <string>
+#include <unordered_map>
+
+#include "onnx_import/core/operator_set.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        /// \brief      Type of container which stores opset version and domain in ONNX format
+        using OpsetImports =
+            ::google::protobuf::RepeatedPtrField<ONNX_NAMESPACE::OperatorSetIdProto>;
+
+        std::string get_node_domain(const ONNX_NAMESPACE::NodeProto& node_proto);
+
+        std::int64_t get_opset_version(const ONNX_NAMESPACE::ModelProto& model_proto,
+                                       const std::string& domain);
+
+        class Model
```

Comment:
Can we merge a separate onnx_import directly to ONNX FE and prepare for onnx_import API deprecation having ONNX FE as a replacement that is aligned with other FWs? In this case this Model class as well as others become a kind of internal API inside ONNX FE.